frontend/config.js

```diff
@@ -3,7 +3,7 @@ const fs = require('fs');
 module.exports = {
   SERVER_PORT: 8000,
   IS_DEBUG: (process.env.NODE_ENV === 'development'),
-  USE_HTTPS: (process.env.USE_HTTPS != 0),
+  USE_HTTPS: (process.env.USE_HTTPS !== '0'),
```

Comment:
This is exactly what I was going to suggest actually but decided not to say anything, glad you came to the same conclusion.